pkg/devserver/service.go

```diff
@@ -324,19 +324,25 @@ func upsertErroredApp(
 		})
 		if err != nil {
 			logger.From(ctx).Error().Err(err).Msg("error inserting app")
+			fmt.Println(1)
```

Comment:
Can we remove these rogue fmt.Printlns?

Reply:
You've discovered my advanced debugging technique